vsintegration/src/FSharp.Editor/SignatureHelp.fs

```diff
@@ -155,17 +155,22 @@ type FSharpSignatureHelpProvider [<ImportingConstructor>]  (serviceProvider: SVs
 
         for method in methods do
             // Create the documentation. Note, do this on the background thread, since doing it in the documentationBuild fails to build the XML index
-            let methodDocs = XmlDocumentation.BuildMethodOverloadTipText(documentationBuilder, method.Description, true)
+            let methodDocs = XmlDocumentation.BuildMethodOverloadTipText(documentationBuilder, method.Description, false)
```

Comment:
I believe we still want true here

* with "true" the active parameter is shown underneath the list of all parameters. The active parameter is shown in bold. There is duplication.

* with "false", only one the active parameter is shown, no others

The duplication is unfortunate and seems a Roslyn proble? But I think we definitely want all parameters - more information is better than less here.